sys/net/gnrc/network_layer/ipv6/nib/nib.c

```diff
@@ -1732,4 +1744,59 @@ static uint32_t _handle_rio(gnrc_netif_t *netif, const ipv6_hdr_t *ipv6,
 
     return route_ltime;
 }
+
+#if IS_ACTIVE(CONFIG_GNRC_IPV6_NIB_QUEUE_PKT)
+gnrc_pktqueue_t *_nbr_pop_pkt(_nib_onl_entry_t *node)
+{
+    if (node->pktqueue_len == 0) {
+        assert(node->pktqueue == NULL);
+        return NULL;
+    }
+    assert(node->pktqueue != NULL);
+
+    node->pktqueue_len--;
+
+    return gnrc_pktqueue_remove_head(&node->pktqueue);
+}
+
+void _nbr_push_pkt(_nib_onl_entry_t *node, gnrc_pktqueue_t *pkt)
+{
+    assert(node->pktqueue_len <= CONFIG_GNRC_IPV6_NIB_QUEUE_PKT_CAP);
+    if (node->pktqueue_len == CONFIG_GNRC_IPV6_NIB_QUEUE_PKT_CAP) {
+        gnrc_pktqueue_t *oldest = _nbr_pop_pkt(node);
+        assert(oldest != NULL);
+        gnrc_pktbuf_release_error(oldest->pkt, EHOSTUNREACH);
+        pkt->pkt = NULL;
```

Comment:
What does this do?

Reply:
This is how a `gnrc_pktqueue_t` entry is marked as free, and should be 
```c
oldest->pkt = NULL;
```
Good catch, I'll fix this next week!

Reply:
Fixed!